.travis.yml

```diff
@@ -28,9 +28,7 @@ matrix:
 
 before_install:
     - composer self-update
-    - if [ "$SYMFONY_VERSION" != "" ]; then
```

Comment:
What's the issue? Looks far more readable to have in on multiple lines IMO. And Insight YAML checker doesn't apply to the `.travis.yml` as Travis is parsing the YAML a different way.